gw_eccentricity/load_data.py

```diff
@@ -174,22 +194,16 @@ def load_LAL_waveform(**kwargs):
         default is "EccentricTD".
```

Comment:
What about this? Please do another read of the docs in this script to make sure everything is good.

Reply:
A quick scan of the code is probably good. For example, we should not be saying "Stupid EccentricTD only works for finite ecc" in public code.